{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestComplexScenario"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "22856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608286057655, "uuid": "68508fe9-8412-4e30-aae4-df5aff67e745", "historyId": "6df5cbf7c00ce5f14b4c42d69bcd2de9", "name": "test_verification_of_pick_action", "start": 1608286050477, "fullName": "tests.complex_test.TestComplexScenario#test_verification_of_pick_action", "statusDetails": {"message": "RuntimeError: No Trajectory execution capability available.", "trace": "self = <tests.complex_test.TestComplexScenario instance at 0x7f1a0d649f38>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        mo = Model('glass')\n        data_logger('logger/logs/pick_action_start')\n        hx,hy,hz = mo.lucy_pos()[0],mo.lucy_pos()[1],mo.lucy_pos()[2]\n        # Pick from table and place on shelf\n        result = MoveItPickAndPlace( pick_x = 0, pick_y = 3.4 , pick_z = 0.54,\n>                                    place_x = 0.5, place_y = 3.5 , place_z = 0.52)\n\ntests/complex_test.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/action_client/pick_place_client.py:29: in __init__\n    base = moveit_commander.MoveGroupCommander(\"base\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <moveit_commander.move_group.MoveGroupCommander object at 0x7f1a0cbad4d0>\nname = 'base', robot_description = 'robot_description', ns = ''\n\n    def __init__(self, name, robot_description=\"robot_description\", ns=\"\"):\n        \"\"\" Specify the group name for which to construct this commander instance. Throws an exception if there is an initialization error. \"\"\"\n>       self._g = _moveit_move_group_interface.MoveGroupInterface(name, robot_description, ns)\nE       RuntimeError: No Trajectory execution capability available.\n\n/opt/ros/kinetic/lib/python2.7/dist-packages/moveit_commander/move_group.py:52: RuntimeError"}, "attachments": [{"source": "ccc881c0-de5e-4344-99bf-e32a45ced797-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "ebb392c8-f463-4715-bc7d-f1132e2ae7d1-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "7ac6d796-8ef7-407c-8933-6583dfa84f50-attachment.txt", "type": "text/plain", "name": "stderr"}]}